openapi-cli/src/main/resources/templates/utils.bal

```diff
@@ -233,6 +233,14 @@ isolated function getMapForHeaders(map<any> headerParam) returns map<string|stri
     foreach var [key, value] in headerParam.entries() {
         if value is string || value is string[] {
             headerMap[key] = value;
+        } else if value is int[] {
+            string[] stringArray = [];
+            foreach int intValue in value {
+               stringArray.push(intValue.toString());
+            }
+            headerMap[key] = stringArray;
+        } else if value is int || value is boolean {
```

Comment:
```suggestion
        } else if value is SimpleBasicType {
```
I think for any primitive type we could simply use `toString()` to attach in `headerMap`. Was there any specific reason to select only `int` and `boolean` ?

Reply:
Yes. Addressed in https://github.com/ballerina-platform/ballerina-openapi/commit/42d8cb145f00ec8152ad6c2d6d3dd9b1d2909dcb